File: StructuralDesignPattern/AdapterPattern/java/Driver.java
package StructuralDesignPattern.AdapterPattern.java;

import StructuralDesignPattern.AdapterPattern.java.AbstractTarget.MediaPlayer;
import StructuralDesignPattern.AdapterPattern.java.ConcreteTarget.Mp3Player;

public class Driver {
    public static void main(String[] args){
        MediaPlayer audioPlayer = new Mp3Player();

        audioPlayer.play("mp3", "song.mp3");
        audioPlayer.play("mp4", "movie.mp4");
        audioPlayer.play("vlc", "video.vlc");
        audioPlayer.play("avi", "random.avi");
    }
}

File: StructuralDesignPattern/AdapterPattern/python/Driver.py
from ConcreteTarget.Mp3Player import Mp3Player


if __name__ == "__main__":

    mp3Player = Mp3Player()
    mp3Player.play("mp3", "mp3file")
    mp3Player.play("mp4", "mp4file")
    mp3Player.play("vlc", "vlcfile")
    mp3Player.play("avi", "avifile")